R/run_prep_calculation_loans.R

```diff
@@ -1,13 +1,20 @@
 #' Run stress testing data preparation for loans
 #'
+#'
 #' @inheritParams run_stress_test
+#' @param data_prep_output_path Path where results are written.
+#'   NOTE: This is a workflow that is needed exclusively in preparation of
+#'   [run_stress_test()] for loan books. It creates input data for the stress
+#'   test. A recommended setting is to set `data_prep_output_path` to to the same
+#'   path as `input_path_project_specific`.
 #' @param credit_type Type of credit. For accepted values please compare
 #'   `credit_type_lookup`.
 #'
 #' @return NULL
 #' @export
-run_prep_calculation_loans <- function(data_path_project_specific,
-                                       data_path_project_agnostic,
+run_prep_calculation_loans <- function(input_path_project_specific,
+                                       input_path_project_agnostic,
+                                       data_prep_output_path,
```

Comment:
Here is one example of the long-name problem. The wrapping is less clear than this -- I believe:

```r
prep_loans <- function(general_input_path, specific_input_path, output_path) { ...
```

And it would be much clearer if you make each function do one thing and only one thing well:

```r
loans_read <- function(general_path, specific_path) { ...

loans_prep <- function(data) { ...

loans_write <- function(data, path) { ...
```

Note here the `loan_` prefix plays well with autocomplete and makes a clear family versus `st_*()` functions.